### PR TITLE
update to latest TP SDK

### PR DESCRIPTION
**Pull request by dsyme**
This is a trial to update to the latest Type Provider SDK

### CODE REVIEW

**Comment by dsyme** [comment]
I believe this to be safe to integrate once CI is green

**Comment by dsyme** [comment]
Just to say I got a WorldBank  error when running tests locally: 

image

**Comment by ovatsus** [comment]
WorldBank keeps changing, so it doesn't even run on CI

File: tests/FSharp.Data.DesignTime.Tests/SignatureTests.fs [hunk 1]
```diff
@@ -39,37 +40,88 @@ let portableRuntimeAssembly profile = sourceDirectory ++ ".." ++ ".." ++ "bin" +
...
+/// Simulate a real host of TypeProviderConfig
+type internal TcImports(bas: TcImports option, dllInfos: DllInfo list) =
+    member x.Base = bas
+    member x.DllInfos = dllInfos
+
+
+type internal Testing() =
+
+    /// Simulates a real instance of TypeProviderConfig
+    static member MakeSimulatedTypeProviderConfig (resolutionFolder: string, runtimeAssembly: string, runtimeAssemblyRefs: string list) =
```
Comment (ovatsus) [inline tests/FSharp.Data.DesignTime.Tests/SignatureTests.fs]: Can you move this to TypeProviderInstantiation.fs so it also works on Test.fsx?

Comment (dsyme) [reply tests/FSharp.Data.DesignTime.Tests/SignatureTests.fs]: @ovatsus I removed that code, thanks

**Comment by dsyme** [comment]
@ovatsus This should be ready now.  Travis CI is also back up and green after fixes in the type provider SDK testing apparatus for finding suitable references!

**Comment by ovatsus** [comment]
So does this change make FSharp.Data support .Net Standard 2.0 or .Net Core 2.0 or something like that? Sorry but I'm really not up to date with all those new things

**Comment by dsyme** [comment]
> So does this change make FSharp.Data support .Net Standard 2.0 or .Net Core 2.0 or something like that? Sorry but I'm really not up to date with all those new things

No, it actually doesn't add anything new for FSharp.Data (apart from getting Travis CI green), which was already using the ProvidedContext facility to allow multi-targeting.  It's just a sanity check that the updates to the TPSDK are ok.   The latest version of TPSDK makes it compulsory to use ProvidedContext

**Comment by realvictorprm** [comment]
Awesome :) Thanks for this!

Am 10.10.2017 11:20 nachm. schrieb "Don Syme" <notifications@github.com>:

**Pull request merged**